src/FNAPlatform/OpenGLDevice.cs

```diff
@@ -663,6 +647,17 @@ ref wmInfo
 			MojoShader.MOJOSHADER_glMakeContextCurrent(shaderContext);
 			FNALoggerEXT.LogInfo("MojoShader Profile: " + shaderProfile);
 
+			// UIKit needs special treatment for backbuffer behavior
+			if (SDL.SDL_GetPlatform() == "iOS" || SDL.SDL_GetPlatform() == "tvOS")
```

Comment:
Was the wminfo data inaccurate? If possible I'd like to get the GLuint handles from SysWMInfo instead, but if something's not right with it then we can do this.

Reply:
Yeah, it always returned 0 for both the Framebuffer and Renderbuffer handles, which just put us back where we started.

Reply:
I wonder if it’s this part here: https://hg.libsdl.org/SDL/file/80f861112154/src/video/uikit/SDL_uikitwindow.m#l357

Depending on the reason for these results this may be an SDL bug.